carbon-design-system/carbon
# fix(Combobox): add event propagation to mouse down

Closes #9073 

- removes `event.stopPropagation()` from the `onMouseUp` event on the toggle button's props.
- `onMouseUp` on the toggle button now moves active focus to the text input

#### Testing / Reviewing

- go into `../ComboBox-story.js` and duplicate the `<ComboBox>` seen on line 65 like so
![image](https://user-images.githubusercontent.com/40970507/128085933-75406b2d-0e5a-45d2-a56c-586b3f997872.png)
- open the **second** ComboBox's listbox either by clicking the text input or the toggle button
- click the **first** ComboBox's toggle button and observe the **second** Combobox's listbox close as expected

#### Notes
@joshblack pretty sure stopping event propagation was critical for some reason we're missing here -- do you remember why?
